standard/standard_4.php

```diff
@@ -1161,4 +1151,4 @@ function getmxrr ($hostname, array &$mxhosts, array &$weight = null) {}
  * </table>
  * @since 5.0
  */
-function dns_get_record ($hostname, $type = DNS_ANY, array &$authns = null, array &$addtl = null, &$raw = false) {}
\ No newline at end of file
+function dns_get_record ($hostname, $type = DNS_ANY, array &$authns = null, array &$addtl = null, &$raw = false) {}
```

Comment:
Please, revert formatting here. The reason is that we want to preserve changes history

Reply:
It looks like GitHub is inserting this automatically :(